config/routes.rb

```diff
@@ -652,6 +652,14 @@
   end
   ####
 
+  #### Admin::Users::SignIns controller
+  namespace :admin do
```

Comment:
Couldn't quite decide on how to route this.

Alternative was…

```ruby
namespace :admin do
  namespace: :users do
    resources :sign_ins, only: [:index] #=> /admin/users/sign_ins
  end
end
```

…but wanted to avoid nesting hell. Could possibly do the above and use shallow routes if more actions added. Open to suggestion.